release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md

```diff
@@ -76,9 +76,86 @@ To view bug fixes, see the [GitHub milestone for 2201.2.0 (Swan Lake)](https://g
 
 ### Runtime updates
 
+#### New features
+
+##### Strand dump
+
+Allows to get the status of strands and strand groups during the execution of a Ballerina program.
+
+This can be used to troubleshoot runtime errors. Ballerina runtime will emit the strand dump to the standard output stream in the text format, if it receives a `SIGTRAP` signal. (`SIGTRAP` is not available on Windows.)
+
+e.g., If the PID of the running Ballerina program is `$PID`, you can get the strand dump by executing either `kill -SIGTRAP $PID` or `kill -5 $PID` command.
+
+##### StopHandler
```

Comment:
```suggestion
##### `StopHandler`
```
Shall we specify what this is (i.e., `StopHandler` function or module likewise)?